RelationalAI/Models/Engine/Engine.cs

```diff
@@ -41,8 +41,7 @@ public class Engine : Entity
         public string DeletedOn { get; set; }
 
         [JsonProperty("size")]
-        [JsonConverter(typeof(StringEnumConverter))]
-        public EngineSize Size { get; set; }
```

Comment:
Could you please fix protobuf dependency issue first and publish it, and then do this change separately? SAM team is going to be using the version with enum here.

Reply:
Sure thing :pray: